src/http/api/contributors.js

```diff
@@ -10,10 +10,12 @@ module.exports = class Contributors extends Route {
 
   register (app) {
     const router = Router()
-
+    // what the fuck is happening here
+    // TODO: everything, it sucks
     router.get('/', async (req, res) => {
       const guild = this.client.guilds.cache.get(process.env.BOT_GUILD)
-      const roles = guild.roles
+      const roles = guild.roles.cache
+      console.log(roles)
```

Comment:
Someone forgot to remove console log?

Reply:
that part of the code still needs to be refactored to support sharding

Reply:
check line 13